app/models/agents/twitter_stream_agent.rb

```diff
@@ -221,8 +229,7 @@ def handle_status(status)
         return if status.has_key?('delete')
         return unless status['text']
         status['text'] = status['text'].gsub(/&lt;/, "<").gsub(/&gt;/, ">").gsub(/[\t\n\r]/, '  ')
-
-        if status["retweeted_status"].present? && status["retweeted_status"].is_a?(Hash)
+        if agent.options[:include_retweets] == "false" && status["retweeted_status"].present? && status["retweeted_status"].is_a?(Hash)
```

Comment:
We need to invert the check to keep the default of `include_retweets = false` for existing Agents. `if agent.options[:include_retweets] != "true" && ..` should do the trick.